src/Provider/Gitlab.php

```diff
@@ -82,18 +82,29 @@ public function getResourceOwnerDetailsUrl(AccessToken $token)
     }
 
     /**
-     * Get the default scopes used by this provider.
+     * Get the default scopes used by GitLab.
+     * Current scopes are 'api', 'read_user', 'openid'
      *
-     * This returns an empty array as Gitlab does not support scopes yet.
+     * This returns an array with 'api' scope as default.
      *
      * @return array
      */
     protected function getDefaultScopes()
     {
-        return [];
+        return ['api'];
     }
 
     /**
+     * GitLab uses a space to separate scopes
+     */
+    protected function getScopeSeparator()
+    {
+        return ' ';
+    }
+
+
+
+    /**
```

Comment:
Given that it's a GitLab constant in a GitLab class I'd prefer it to be a true `CONST` declaration on top of the class.